app/assets/src/components/Samples.jsx

```diff
@@ -858,6 +860,27 @@ class Samples extends React.Component {
     }
   }
 
+  handleCreateBackground(name, description, sample_ids) {
+    var that = this;
+    axios
+      .post("/backgrounds", {
+        name: name,
+        description: description,
+        sample_ids: sample_ids,
+        authenticity_token: this.csrf
+      })
+      .then(response => {
+        that.setState({
+          background_creation_response: response.data
+        });
+      })
+      .catch(error => {
```

Comment:
Shouldnt we give some feedback to the user?

Reply:
yes, will add